.github/workflows/validate-snapshots.yml

```diff
@@ -2,7 +2,9 @@ name: Validate snapshots
 on:
   workflow_dispatch:
   schedule: 
-    - cron: '35 9,12,16 * * 1-5'
+    # Action runs 1 hour ahead of GMT
+    # This action runs at 9:35am, 12am, 4pm UK time
+    - cron: '35 8,11,15 * * 1-5' 
```

Comment:
Doesn't look like there's any support currently for timezones and daylight saving time?

Based on the old setting we'd expect the schedule to trigger at:

```
UTC/GMT: 9:35, 12:35, 16:35
BST: 10:35, 13:35, 17:35
```

But the weird thing is according the history of scheduled runs, yesterday it ran at 5:52 GMT (6:52 BST). 😕 

Based on the new setting we'd expect it to be:

```
UTC/GMT: 8:35, 11:35, 15:35
BST: 9:35, 12:35, 16:35
```

Until they add time zone support, to get it to run at 9.35 I think we'd have to change update this config twice a year, swapping it from 9:35 to 8:35 at the beginning of British Summer Time and swapping it back to 9:35 at the end of British Summer Time.

Could add a comment to say this?
Also do you know why it ran at 5:52 GMT (6:52 BST) yesterday? I might be missing something here.

Reply:
Sounds good, I've added a comment around this. I tried to run the snapshot pipeline manually yesterday but Actions had a major outage yesterday which affected the time the pipeline actually ran at.

Reply:
Ok good to know about the outage, thanks!